### PR TITLE
Add directive to ignore i18n

### PR DESCRIPTION
**Pull request by connor4312**
#### PR checklist

- [x] Ran `npm run lint` and `npm run test`
- [x] Added tests if applicable

#### What changes did you make?

```html
<span>
  <!-- i18n-checker disable -->
  support@beam.pro
</span>
```

Ignores the subtree.

### CODE REVIEW

File: src/Validator.ts [hunk 1]
```diff
@@ -22,18 +23,21 @@ export interface II18nValidatorOptions {
...
     assumeTextCondition?: RegExp;
+    ignoreComment?: RegExp;
 }
 
 export class I18nValidator {
 
     public static readonly defaultAttributeMacher = /^([\w-]+)#(\w+):(\w+)\|.*?$/;
     public static readonly defaultTemplateMatcher = /\{\{.*?\}\}/g;
     public static readonly defaultAssumeTextCondition = /\w{2,}/;
+    public static readonly defaultIgnoreComment = /^\s*i18n-checker:? ?disable\s*$/;
```
Comment (SimonSchick) [inline src/Validator.ts]: You could expose this in `bin.ts` as well.
Also can we try to be strict about the whitespaces here? I'd prefer `i18-checker:disable`, no extra spaces (like TSLint).

File: src/Validator.ts [hunk 2]
```diff
@@ -81,10 +86,18 @@ export class I18nValidator {
...
+                oncomment: (comment: string) => {
+                    if (this.options.ignoreComment.test(comment)) {
+                        curr.ignored = true;
+                    }
+                },
                 ontext: (text: string) => {
                     if (curr && this.options.ignoreTags.includes(curr.tag)) {
                         return;
                     }
+                    if (curr && curr.ignored) {
```
Comment (SimonSchick) [inline src/Validator.ts]: We could actually merge together the 3 early returns here, optional tho.

File: src/Validator.ts (hunk 1 above)
Comment (connor4312) [inline src/Validator.ts]: sure, I can tighten it as `( |:)`. My annoyance there is that tslint uses `tslint:disable-line` and eslint uses `eslint disable-line`.

Comment (SimonSchick) [inline src/Validator.ts]: That works, and yea I know, there isn't much of a standard 😛

**Comment by SimonSchick** [comment]
You broke your own test 😛

**Comment by connor4312** [comment]
Shhhh

**Pull request merged**